### ISSUE TITLE
[Adaptive Cards> RestaurantOrder.json]: Screen reader is not announcing the information as required on 'Which entree would you like' edit field.

### ISSUE THREAD

**Issue by vaishali1397** [microsoft/AdaptiveCards#8632]
### Target Platforms

UWP

### SDK Version

1.2.2306.30005

### Application Name

Adaptive Cards

### Problem Description

**Test Environment:**
OS version: 23H2 (Build 25905.1000)
Application: AdaptiveCardVisualizer
Application Version: 1.2.2306.30005
Screen Reader: Narrator

**Repro Steps:**

Open the Adaptive Card Visualizer application.
Turn on the narrator through Ctrl+ Windows+ Enter key.
Navigate to the 'RestaurantOrder.json' through tab key and activate it.
Navigate to 'Which entree would you like' edit field through tab key.

Observe the issue.

**Actual Result:**
Under RestaurantOrder.json card, screen reader is not announcing the information as required on 'Which entree would you like' edit field.
Screen reader is announcing as **'Which entree would you like edit suggestions available'.**

**Expected Result:**
Screen reader should announce the information as required on 'Which entree would you like' edit field.
Screen reader should announce as '**'Which entree would you like edit required suggestions available'.**

**Notes:**
Same issue repro for 'Which side would you like?' and 'Which drink would you like?' edit field.

**Observations:**
Same issue repro with NVDA.

**User Impact:**
Screen redaer user will get impacted as user will not be able to know that this information is required or not and will have difficulty in navigating or in tracking the content.

**Reference link:**
https://www.w3.org/WAI/WCAG21/Understanding/info-and-relationships

“Have feedback to share on Bugs? Please help fill [Trusted Tester Bug Feedback (office.com)](https://nam06.safelinks.protection.outlook.com/?url=https://forms.office.com/pages/responsepage.aspx?id%3Dv4j5cvGGr0GRqy180BHbR5dUqGQgJ9lItv-jqY8F09lURVdYTENIRkY2UzlCR0NKV1E2VUE5OVpDVy4u&data=04%7c01%7cv-vaibgu%40microsoft.com%7c551850210d444be9b28308d9691e3dde%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c637656400980259288%7cUnknown%7cTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7c1000&sdata=TrcBaCyAPKg0WAQfdSZJP2i9pghMEDg59i2NeUUs5EQ%3D&reserved=0) “

https://github.com/microsoft/AdaptiveCards/assets/104483005/7b4f9e89-7f1d-4aa0-96dc-42605125ab18

### Card JSON

```JSON
NA
```

**Comment by vagpt** [comment]
Hi @jwoo-msft,

Could you please look in this issue and provide us ETA  to fix this issue as of now due to this issue Product is getting non-compliance.

**Comment by jwoo-msft** [comment]
> Hi @jwoo-msft,
> 
> Could you please look in this issue and provide us ETA to fix this issue as of now due to this issue Product is getting non-compliance.

I'm assuming the product will be non-compliant if it's not fixed this month?

**Comment by vagpt** [comment]
Adding A11yTTvalidated tag as the UI is not available.
